ajax/kanban.php

```diff
@@ -0,0 +1,215 @@
+<?php
+/**
+ * ---------------------------------------------------------------------
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ * Copyright (C) 2015-2018 Teclib' and contributors.
+ *
+ * http://glpi-project.org
+ *
+ * based on GLPI - Gestionnaire Libre de Parc Informatique
+ * Copyright (C) 2003-2014 by the INDEPNET Development Team.
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * GLPI is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * GLPI is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with GLPI. If not, see <http://www.gnu.org/licenses/>.
+ * ---------------------------------------------------------------------
+ */
+
+$AJAX_INCLUDE = 1;
+
+include ('../inc/includes.php');
+
+header("Content-Type: text/html; charset=UTF-8");
+Html::header_nocache();
+
+Session::checkLoginUser();
+
+if ($_SERVER['REQUEST_METHOD'] !== 'GET') {
+   // Get AJAX input and load it into $_REQUEST
+   $input = file_get_contents('php://input');
+   parse_str($input, $_REQUEST);
+}
+
+if (isset($_REQUEST['action']) && $_REQUEST['action'] === 'get_translated_strings') {
+   header("Content-Type: application/json; charset=UTF-8", true);
+   echo json_encode((Kanban::getLocalizedKanbanStrings()), JSON_FORCE_OBJECT);
+   return;
+}
+
+$nonkanban_actions = ['update', 'add_item'];
+if (!isset($_REQUEST['action']) || !isset($_REQUEST['itemtype']) ||
+   (!in_array($_REQUEST['action'], $nonkanban_actions) && !class_uses($_REQUEST['itemtype'], 'Kanban'))) {
```

Comment:
`class_uses` is not used correclty here.
```suggestion
   (!in_array($_REQUEST['action'], $nonkanban_actions) && !in_array('Kanban', class_uses($_REQUEST['itemtype'])))) {
```